omod/src/test/java/org/openmrs/Allergen.java

```diff
@@ -0,0 +1,7 @@
+package org.openmrs;
+
+/**
+ * Added to ensure that the tests pass
```

Comment:
I comment like this would be more appropriate: https://github.com/openmrs/openmrs-module-idgen/blob/master/omod/src/test/java/org/openmrs/CodedOrFreeText.java#L12

Reply:
@dkayiwa Done